pages/search/index.vue

```diff
@@ -63,15 +68,29 @@
 
     head() {
       return {
-        title: this.$t('search')
+        title: this.$route.query.query ? this.$t('searchResultsFor', [this.$route.query.query]) : this.$t('search')
       };
     },
 
     async beforeRouteLeave(to, from, next) {
+      // Leaving the search page closes the search bar. Reevaluate when autosuggestions go straight to entity pages.
+      this.$store.commit('search/setShowSearchBar', false);
       await this.$store.dispatch('search/deactivate');
       next();
     },
 
     watchQuery: ['api', 'reusability', 'query', 'qf', 'page']
   };
 </script>
+
+<style lang="scss" scoped>
+  h1 {
+    font-size: 1.875rem;
+    font-weight: 300;
+    line-height: 1.375;
+    margin-bottom: 1rem;
+    span {
+      font-weight: 600;
+    }
+  }
```

Comment:
This might have been discussed somewhere, but I think we should keep the same h1 styling on every page. That is better for accessibility too.

Reply:
yeah, this is to align the styling with that of the heading on collection pages where there is already a h1 for the collection title. We could  make it a h2 on the search results page, but then we'd be missing a h1 for the title here altogether.